packages/react-recommendations/babel.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = (api) => {
```

Comment:
We can keep a single Babel config at the monorepo root and provide [`overrides`](https://babeljs.io/docs/en/options#overrides) for each package specificity.

Reply:
Done in https://github.com/algolia/react-recommendations/pull/6/commits/956efe16d5ec7ded275cbac4540230c44dd0904c